tests/v3_api/test_upgrade.py

```diff
@@ -146,6 +162,60 @@ def create_and_validate_ingress_xip_io():
     validate_ingress_xip_io(ingress_name2_create, ingress_wlname2_create)
 
 
+def modify_resources_validate():
+
```

Comment:
@soumyalj Split them into 4 separate calls for each of the scenarios being tested .

Reply:
Made four methods to do these scenarios